packages/esm-patient-medications-app/src/components/medications-details-table.scss

```
@@ -66,3 +66,10 @@
     border-bottom: none !important;
   }
 }
+
+.tag {
+  margin: 0 !important;
+  margin-left: ((layout.$spacing-03)) !important;
```

```suggestion
  margin-left: layout.$spacing-03 !important;
```

We don't need the double parentheses here